pillar/test.sls

```diff
@@ -101,8 +101,32 @@ poff:
 
 
 tls-terminator:
+    error_pages:
+        {% raw %}
+        429: |
+            <!doctype html>
+            <title>That's enough</title>
+            <p>Too much traffic from you to, give the poor server at {{ site }} a rest.</p>
```

Comment:
I like this, but there's seems to be something wrong with the sentence 
`Too much traffic from you to, `

Reply:
Ah, agree, seems like I messed up two different phrasings here at some point.